charmhub/client.go

```diff
@@ -37,9 +39,9 @@ const (
 	CharmhubServerEntity  = "charms"
 )
 
-const (
+var (
 	userAgentKey   = "User-Agent"
-	userAgentValue = "Golang_CSClient/4.0"
+	userAgentValue = "Juju/" + version.Current.String()
```

Comment:
I think the idea is to start sending more fine-grained details about what is asking for the refresh, so this may be shortlived in the end and we might require the user of the client to pass in the agent value.

Reply:
I believe so.